### PR TITLE
Fix doc of clip op.

### LINKED ISSUE #4421: Fix doc of clip op.

### PR DESCRIPTION
**Pull request by wanghaoshuang**
fix #4421

### CODE REVIEW

File: paddle/operators/clip_op.cc [hunk 1]
```diff
@@ -43,7 +45,7 @@ class ClipOpMaker : public framework::OpProtoAndCheckerMaker {
       : OpProtoAndCheckerMaker(proto, op_checker) {
     AddInput("X",
              "(Tensor)The input of clip op."
-             "The input should be a k-D tensor(k > 0 and k < 7)");
+             "The dimension size of input tensor must be between [1, 9].");
```
Comment (typhoonzero) [inline paddle/operators/clip_op.cc]: dimension size => number of dimensions

File: paddle/operators/clip_op.cc [hunk 2]
```diff
@@ -17,6 +17,8 @@
 namespace paddle {
 namespace operators {
 
+using framework::LoDTensor;
```
Comment (typhoonzero) [inline paddle/operators/clip_op.cc]: This line seems not needed?

Comment (wanghaoshuang) [reply paddle/operators/clip_op.cc]: Thx.

File: paddle/operators/clip_op.cc (hunk 1 above)
Comment (wanghaoshuang) [reply paddle/operators/clip_op.cc]: Thx.

**Pull request merged**